applications/tari_validator_node/src/json_rpc/handlers.rs

```diff
@@ -170,9 +170,11 @@ impl JsonRpcHandlers {
         let hash = *mempool_tx.hash();
 
         let subscription = self.hotstuff_events.subscribe();
+        // create a mempool request out of mempool_tx
+        let mempool_request = MempoolRequest::SubmitTransaction(Box::new(mempool_tx));
```

Comment:
In this case, we want to submit a transaction, I think it is safe to assume we have to store it on the mempool, and not have to consider the case in which the transaction is possibly finalized.